hyperspy/models/eelsmodel.py

```diff
@@ -538,31 +538,92 @@ def _fit_edge(self, edgenumber, start_energy=None, **kwargs):
         # Recover the channel_switches. Remove it or make it smarter.
         self.channel_switches = backup_channel_switches
 
-    def quantify(self):
-        """Prints the value of the intensity of all the independent
-        active EELS core loss edges defined in the model
+    def quantify(self, output="absolute"):
```

Comment:
I find the meaning the `output` not clear here. What's about having two argument: `normalise=True/False` and `return_output=True/False` (or `return_results`)?

Reply:
Yeah, I think your suggestion here is better than my implementation. I tried to avoid too many parameters to not "clutter" the code. :-)

Reply:
@ericpre I am back at work and will implement the changes shortly. However, to prevent a misunderstanding:
1) With `return_output` you mean to (optionally) return the dictionary --> This means that the printed output (in the prettytable) is always printed.
Or...
2) With `return_output` you mean to (optionally) print the output --> This will always return the dictionary but will omit the printed output.

I think you meant the first version as currently `quantify` will always print an output, but I just wanted to make sure.

Also, i found a `print_html` function for the prettytable in hyperspy/misc/utils.py which seems to generate the same output as simple `print`. Should I use the `print_html`?

```python
import prettytable
table = PrettyTable()
...
print(table)
#or
print(print_html(f_text=table.get_string,
                          f_html=table.get_html_string))
```

Reply:
I would say that both (displaying and returning results) should be optional. Indeed, currently `quantify` always display the results, but I guess this is a very old behaviour and it would an improvement to be able to disable it, for example to be able to use as part of other programme. Not to break the current behaviour, for now, we should keep the default to display the results.

> Also, i found a `print_html` function for the prettytable in hyperspy/misc/utils.py which seems to generate the same output as simple `print`. Should I use the `print_html`?

Yes, use `print_html`, because running the following in the notebook, doesn't display the html version of the table:
```python
from prettytable import PrettyTable
x = PrettyTable()
x.field_names = ["City name", "Area", "Population", "Annual Rainfall"]
x.add_row(["Adelaide", 1295, 1158259, 600.5])
print(x)
```